spacewalk/certs-tools/spacewalk-certs-tools.changes

```diff
@@ -1,3 +1,4 @@
+- added correct SUMA version in traditional client bootstr. script
```

Comment:
You missed the empty line :-)

Also write `bootstrap` script, the full name :-)

Reply:
Done. Is it okay now? :)